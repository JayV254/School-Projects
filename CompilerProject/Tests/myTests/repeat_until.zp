int main() {
    int a,count;
    a = 1;
    count = 0;
    puts("If you started with 1 dollar and doubled it every day it would take this many day to reach 1 million dollars: ");
    repeat {
        count = count + 1;
        a = 2 * a;
    } until (a > 1000000);
    putn(count);
    return 0;

}
