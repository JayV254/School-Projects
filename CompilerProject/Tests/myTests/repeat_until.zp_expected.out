If you started with 1 dollar and doubled it every day it would take this many day to reach 1 million dollars:     20 

